copier/template.py

```diff
@@ -170,23 +170,32 @@ class Template:
 
             Helpful if you want to test templates before doing a proper release, but you
             need some features that require a proper PEP440 version identifier.
+
+        config_file:
+            Path to the file to be used instead copier.y?ml
```

Comment:
Typo `y?ml`?

Reply:
Yes, it had to be `ya?ml`, but now I have changed the description to be more straightforward